config/cht/branches_with_users.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "id": "d7302e6b-ba71-4463-85e4-add3eaae69ce",
+    "name": "Test Health Facility",
+    "skip": false,
+    "users": {
```

Comment:
A comment about the location of these files. Is `cht` the right name for this folder? Should we call it something like `config/example`?

Reply:
I kept it as `cht` because I used `instance=cht.app` to upload through `medic-conf` and didn't want to see the warning of non-matching instance. As we can upload by specifying URL, I am open to change it to whatever seems reasonable.

Reply:
`cht` just seems a bit odd because the `webapp` respository is all part of the `cht`.  Some sort of specific folder name would be more clear. Maybe a good question for the CHT working group? Or `config/reference`?

Reply:
I agree, it would be nice to have a more descriptive name. Given that this will be the default config as part of https://github.com/medic/medic/issues/5914 should we name it `config/default`?

There will be other reference, example, and base configs in the future, but only one default.

@garethbowen, thoughts?

Reply:
I like `config/default`.